docs/manage-model-metadata-using-gravitino.md

```diff
@@ -0,0 +1,637 @@
+// ...
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+gravitino_client: GravitinoClient = GravitinoClient(uri="http://localhost:8090", metalake_name="example")
+
+catalog: Catalog = gravitino_client.load_catalog(name="model_catalog")
+model: Model = catalog.as_model_catalog().get_model(ident=NameIdentifier.of("model_schema", "example_model"))
+```
+
+</TabItem>
+</Tabs>
+
+### Delete a model
+
+You can delete a model by sending a `DELETE` request to the `/api/metalakes/{metalake_name}
+/catalogs/{catalog_name}/schemas/{schema_name}/models/{model_name}` endpoint or by using the
+Gravitino Java/Python client. The following is an example of deleting a model:
+
+<Tabs groupId="language" queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X DELETE -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" \
+http://localhost:8090/api/metalakes/example/catalogs/model_catalog/schemas/model_schema/models/example_model
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// ...
+Catalog catalog = gravitinoClient.loadCatalog("model_catalog");
+catalog.asModelCatalog().deleteModel(NameIdentifier.of("model_schema", "example_model"));
+// ...
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+gravitino_client: GravitinoClient = GravitinoClient(uri="http://localhost:8090", metalake_name="example")
+
+catalog: Catalog = gravitino_client.load_catalog(name="model_catalog")
+catalog.as_model_catalog().delete_model(NameIdentifier.of("model_schema", "example_model"))
+```
+
+</TabItem>
+</Tabs>
+
+Note that the delete operation will delete all the model versions under this model.
+
+### List models
+
+You can list all the models in a schema by sending a `GET` request to the `/api/metalakes/
+{metalake_name}/catalogs/{catalog_name}/schemas/{schema_name}/models` endpoint or by using the
+Gravitino Java/Python client. The following is an example of listing all the models in a schema:
+
+<Tabs groupId="language" queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X GET -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" \
+http://localhost:8090/api/metalakes/example/catalogs/model_catalog/schemas/model_schema/models
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// ...
+Catalog catalog = gravitinoClient.loadCatalog("model_catalog");
+NameIdentifier[] identifiers = catalog.asModelCatalog().listModels(Namespace.of("model_schema"));
+// ...
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+gravitino_client: GravitinoClient = GravitinoClient(uri="http://localhost:8090", metalake_name="example")
+
+catalog: Catalog = gravitino_client.load_catalog(name="model_catalog")
+model_list = catalog.as_model_catalog().list_models(namespace=Namespace.of("model_schema")))
+```
+
+</TabItem>
+</Tabs>
+
+## ModelVersion operations
```

Comment:
Where is the definition of term `ModelVersion`?

Reply:
It is defined in the `model-catalog.md`.